### ISSUE TITLE
Data ...

### ISSUE THREAD

**Issue by amitc786** [Arkoprabho/NSEToolsPy#2]
Hi,
This is not an issues first of all...
My question is as you are taking data from nse's website, is that legal...????

**Comment by Arkoprabho** [comment]
Nope. It isn't legal. And that is why I have stopped working on it. Unfortunately I realized the legality terms after starting the project. I had to abandon it after I realized that it's illegal.
On the flipside, creating the API does not seem to be illegal, using it is. The legal terms say that 

> You may not conduct any systematic or automated data collection activities (including scraping, data mining, data extraction and data harvesting) on or in relation to our website without our express written consent.

This as far as I understand prevents us from actively scraping the website to extract data. Now we as developers are just providing the tool to do so. Whether or not you use this to scrape is on you.
I am no legal expert, but till I can get the legal aspects sorted, I will not work on this API. On that note, I have contacted them through e-mail, and in person. And have very politely been ignored.
